src/main/java/com/code_intelligence/jazzer/driver/Opt.java

```diff
@@ -61,6 +61,8 @@ public final class Opt {
         "Additional arguments to pass to the JVM (separator can be escaped with '\\', native launcher only)");
     stringSetting(
         "agent_path", null, "Custom path to jazzer_agent_deploy.jar (native launcher only)");
+    stringSetting("init_options", null,
```

Comment:
Can we prefix this option with `android_` to make it clear that it is only relevant in this scenario? 

Could you also group the Android settings in a separate block in this file?

Reply:
Done & Done. Please review.